transforms/replace-collapsible-pod-with-accordion/README.md

```diff
@@ -0,0 +1,51 @@
+# replace-collapsible-pod-with-accordion
+
+The `collapsed` prop of the Pod component has been deprecated in favour of placing an Accordion inside the Pod Content. This codemod removes all `collapsed` prop instances and places an `Accordion` inside the Pod Content. Title prop will be also moved to replicate previous functionality.
+
+```diff
+import Pod from "carbon-react/lib/components/pod";
+
++ import Accordion from "carbon-react/lib/components/accordion";
+
+- <Pod title="Title" subtitle="subtitle" collapsed>pod content</Pod>
++ <Pod><Accordion borders="none" expanded title="Title" subtitle="subtitle">pod content</Accordion></Pod>
+```
+
+It's likely that props might be assigned in a different manners, therefore this codemod accounts for several prop patterns.
+
+```js
+<Pod prop="collapsed" title="Title" subtitle="subtitle" />
+```
+
+```js
+const collapsed = true;
+const title = "Title";
+
+<Pod collapsed={collapsed} title={title} subtitle="subtitle" />;
+```
+
+```js
+const props = { prop: "collapsed", title: "Title", subtitle: "subtitle" };
+
+<Pod {...props} />;
+```
+
+```js
+<Pod {...{ prop: "collapsed", title: "Title", subtitle: "subtitle" }} />
+```
+
+```js
+const value = "collapsed";
+
+<Pod {...{ prop: value }} />;
+```
+
+If there is a pattern that you use that is not transformed, please file a feature request.
+
+## Usage
+
+`npx carbon-codemod replace-collapsible-pod-with-accordion <target>`
```

Comment:
This need to be added to `bin/cli.js`. It doesn't work at the moment so I was unable to test it locally other than reviewing the tests.

Reply:
I knew I'll miss out something :)